### PR TITLE
Amend Load Validations - V3

### PR DESCRIPTION
**Pull request by luke-hill**
Remove default load validation and configuration options.

I've opted not to go down the deprecation route, as this is something I only added in as a configuration option to fix the issue of URL-less pages.

### CODE REVIEW

**Comment by luke-hill** [comment]
ping @tgaff @RustyNail <- If you're interested in reviewing stuff for v3

File: spec/loadable_spec.rb [hunk 1]
```diff
@@ -41,33 +41,11 @@
...
-        it 'has no default load validations' do
-          klass = Class.new(SitePrism::Page)
-
-          expect(klass.load_validations.length).to eq(0)
-        end
-      end
-    end
+    context 'A standard Page' do
+      it 'has no default load validations' do
+        klass = Class.new(SitePrism::Page)
```
Comment (ineverov) [inline spec/loadable_spec.rb]: hah! You said you prefer other way of doing this some time [ago](https://github.com/natritmeyer/site_prism/pull/250#discussion_r186646007) 😆 
North remembers!

Comment (luke-hill) [inline spec/loadable_spec.rb]: This area I'm not entirely sure how to get it working the way I want. If you can point me in the right way I'll fix it up.

It "may" be that I can actually fix this now there is only one code path not 2 (0 / 1 default validations). For some of the Loadable tests here, they create phantom classes that just inherit the Loadable module (And are several years old), so I'm loathe to change them.

I'll have a look next week to see if I can standardise this.

**Comment by luke-hill** [comment]
ping @ineverov / @tgaff - I have ...
- Rebased this against the latest v3 alpha work.
- Fixed up documentation / CR comments
- Tidied up some linelength stuff in readme

Let me know what you think. I'll leave it in PR for a couple of weeks so you've got time to look!

File: README.md [hunk 2]
```diff
@@ -1465,14 +1465,13 @@ end
...
-3. The `FooPage` load validation will wait for the `form` element to be present.
-4. The `FooPage` load validation will wait for the `some_other_element` element to be present.
+1. The `BasePage` load validation will wait for the loading message to disappear.
+2. The `FooPage` load validation will wait for the `form` element to be present.
+3. The `FooPage` load validation will wait for the `some_other_element` element to be present.
 
-NOTE: `SitePrism::Page` includes a default load validation on `page.displayed?` which is applied
-to all pages.  It is therefore not necessary to define a load validation for this condition on
-inheriting page objects.
+**NB:** `SitePrism::Page` **used to** include a default load validation on `page.displayed?` however for
```
Comment (ineverov) [inline README.md]: 👍

**Pull request merged**